File: LeetCode/cpp/509.cpp
class Solution {
public:
    int fib(int N) {
        if(N == 0)
            return 0;
        int i = 0, j = 1;
        while (N > 1) {
            int temp = i + j;
            i = j;
            j = temp;
            --N;
        }
        return j;
    }
};

File: LeetCode/python3/509.py
class Solution:
    def fib(self, N):
        """
        :type N: int
        :rtype: int
        """ 
        if N == 0:
            return 0
        a, b = 0, 1
        for i in range(1, N):
            a, b = b, a+b
        return b
